internal/config/main_test.go

```diff
@@ -80,7 +80,7 @@ func TestAutoDetectInBackground(t *testing.T) {
 	}
 	cfg := config.New(
 		config.WithAutoDetect(mock),
-		config.WithAutoDetectFrequency(500*time.Second),
```

Comment:
idk why it was like this lol

Reply:
Timings in test are.. Maybe to prevent it from some flakiness?

Reply:
yeah no idea, it was just breaking things locally for me too